### DISCUSSION TITLE
Implementation for Original Xbox

### DISCUSSION THREAD

**Discussion by colton-everett** [pmgducati/Blue-Retro-AIO-Units#80] [General]
I recently ordered materials to start building an AIO for my setup. I've noticed the glaring omission of an Original Xbox PCB. From my searching, it seems that  an Xbox implementation has been difficult since the controller ports are basically USB. I was wondering if that same limitation applies to the AIO kit. I was optimistic when I saw darthcloud releasing the BRX, but I assume that's the closest we're going to get?

**Comment by pmgducati** [comment]
You need to take that up with darthcloud in the blue retro repo.  I can only make AIO units that the firmware allows for.

**Reply by colton-everett** [reply]
Makes sense. Thanks!
